src/cmap/auth/mongodb_oidc/callback_workflow.ts

```diff
@@ -156,11 +96,10 @@ export class CallbackWorkflow implements Workflow {
   private async startAuthentication(
```

Comment:
vscode says this is dead code.  Can we remove this?

Reply:
It's actually used now in the human callback.